### PR TITLE
Create simple window and MVP structure

### PR DESCRIPTION
**Pull request by ghost**
# Short description of the changes:

Adds MVP structure and an empty home tab

# Long description of the changes:

# Check list for the pull request
- [ ] I have read the [CONTRIBUTING]
- [ ] I have read the [CODE_OF_CONDUCT]
- [ ] I have added tests for my changes
- [ ] I have updated the documentation accordingly

# Check list for the reviewer
- [ ] I have read the [CONTRIBUTING]
- [ ] I have verified the proposed changes
- [ ] best software practices
    + [ ] all internal functions have an underbar, as is python standard
    + [ ] clearly named variables (better to be verbose in variable names)
    + [ ] code comments explaining the intent of code blocks
- [ ] All the tests are passing
- [ ] The documentation is up to date
- [ ] code comments added when explaining intent

# Manual test for the reviewer

# References

### CODE REVIEW

File: src/packagenamepy/__init__.py [hunk 1]
```diff
@@ -1,5 +1,12 @@
...
-try:
-    from ._version import __version__  # noqa: F401
-except ImportError:
-    __version__ = "unknown"
+"""
+Contains the entry point for the application
+"""
+
+from ._version import __version__  # noqa: F401
+
```
Comment (KedoKudo) [inline src/packagenamepy/__init__.py:6]: what is the reason for dropping try-except block here?  `._version` is a generated file, and it should not be part of git.

File: README.md [hunk 2]
```diff
@@ -12,28 +12,62 @@ After you create a new repository using this repo as template, please follow the
...
+    8.4 We strongly recommended using a single `pyproject.toml` file to manage all the project metadata, including the project name, version, author, license, etc.
+
+    8.5 Python is moving away from `setup.cfg`/`setup.py`, and we would like to follow the trend for our new projects.
+
+
+9. Specify package name at  https://github.com/neutrons/python_project_template/tree/df49fa434591c073fe88f50d419b14c789fb1e17/src/packagenamepy
+
+
+10. Adjust 'src/packagenamepy/__main__.py: https://github.com/neutrons/python_project_template/blob/df49fa434591c073fe88f50d419b14c789fb1e17/src/packagenamepy/__main__.py
+
```
Comment (KedoKudo) [inline README.md:67]: please see my comments on #5 about using permanent link in readme.

**Review by KedoKudo — CHANGES_REQUESTED** [review]
In the readme, please add a new section to list the files that needs to be deleted if the package does not have a GUI component.

File: src/packagenamepy/__init__.py (hunk 1 above)
Comment (ktactac-ornl) [reply src/packagenamepy/__init__.py]: Don't know why that got removed

File: README.md (hunk 2 above)
Comment (ktactac-ornl) [reply README.md:67]: Working backwards on that, up next

**Review by ghost — COMMENTED** [review]
updating readme in other branch

**Review by KedoKudo — APPROVED** [review]

**Pull request merged**